### DISCUSSION TITLE
Get vertical swiper to return to slide one when the parent horizontal slider goes to next or previous slide?

### DISCUSSION THREAD

**Discussion by ShaneSheppard** [nolimits4web/swiper#7098] [Q&A]
Not sure where to post this (hope it's the correct place)  but i thought ether way it could use some clarification.

I have a horizontal swiper (parent) and some of the slides for said swiper are vertical swipers (child) inside a horizontal slide.
I would like the (child) vertical swipers to return to the first slide when outside of the viewport for example the next or previous slide on the parent horizontal swiper.

The best example would be this swiperjs demo - 
https://codesandbox.io/p/sandbox/4c7l6v?file=/index.html

How could i make horizontal slide 2 return to vertical slide 1 when viewing horizontal slide 1 or 3?
Is there an easy function to accomplish this that i have missed?

Apologies if this has already been answered but i could not find any clear guidance while searching.

**Comment by Ibrahimsalman01** [comment]
Apologies for reviving this post years later, but were you able to find a solution? I have a similar setup with a parent vertical swiper and multiple child horizontal swipers. I can't figure out how to reset the child's slide position after losing focus.
